### ISSUE TITLE
The tabs slider does not hide.

### ISSUE THREAD

**Issue by wSLecHayfIeNdock** [ant-design/ant-design#8861]
### Version
3.1.0

### Environment
Google Chrome 63

### Reproduction link
[https://codesandbox.io/s/2wpj016070](https://codesandbox.io/s/2wpj016070)

### Steps to reproduce
Container style set to:
```
const style = {
  width: 260,
  overflowY: 'auto',
  position: 'fixed',
  top: 0,
  bottom: 0
}
```
![gif](https://user-images.githubusercontent.com/16831715/34659192-0080eb4e-f471-11e7-8880-0b2cd9d0e87f.gif)

*就是内容超出容器出现滚动条时，Tabs 宽度不够会出现滑块，宽度满足 Tabs 滑块不会隐藏
这里为了简便，所以拉了 console，以致于高度改变触发问题重现*

Even though hide
```
.ant-tabs-nav-container-scrolling {
  padding: 0;
}
.ant-tabs-tab-arrow-show {
  display: none;
}
```

### What is expected?
The slider is hidden when the container is wide enough.

### What is actually happening?
Slider not hide.

**Comment by afc163** [comment]
Fixed in rc-tabs@9.2.3, try reinstall node_modules.

**Comment by miteshtagadiya** [comment]
I have a similar issue, I used vertical tabs and  I used this custom sass, and it works for me,
  ```  .ant-tabs-nav-container-scrolling 
        padding: 0px !important

    .ant-tabs-tab-arrow-show 
        display: none
```
this CSS hides the arrows and then removes the top padding of the arrow.
